pyanaconda/modules/storage/bootloader/base.py

```diff
@@ -725,7 +743,7 @@ def _set_storage_boot_args(self, storage):
         # FIPS
         boot_device = storage.mountpoints.get("/boot")
         if kernel_arguments.get("fips") == "1" and boot_device:
-            self.boot_args.add("boot=%s" % self.stage2_device.fstab_spec)
+            self.boot_args.update(["boot=%s" % self.stage2_device.fstab_spec])
```

Comment:
Please use the `add` method when it's available now.

Reply:
Done! Thanks for catching these.